ocs_ci/utility/prometheus.py

```diff
@@ -13,6 +13,53 @@
 logger = logging.getLogger(name=__file__)
 
 
+def check_alert_list(
+    label,
+    msg,
+    alerts,
+    states,
+    severity="warning"
+):
+    """
+    Check list of alerts that there are 2 alerts with requested label and
+    message. First alerts should be pending and the second alert should be
+    firing.
```

Comment:
This could be tweaked a bit to take into account `states` list with expected states of alerts. I mean this is not about just 2 alerts, but it depends what you put into `states`.